locales/en-US/strings.properties

```diff
@@ -88,25 +88,6 @@ prefs_topstories_sponsored_learn_more=Learn more
 prefs_highlights_description=A selection of sites that you’ve saved or visited
 prefs_snippets_description=Updates from Mozilla and Firefox
 settings_pane_button_label=Customize your New Tab page
-settings_pane_header=New Tab Preferences
```

Comment:
These strings can't be removed yet as we will likely uplift strings to beta

Reply:
Post a comment to https://bugzilla.mozilla.org/show_bug.cgi?id=1445090 with these string ids that should be removed.

Reply:
How about pocket_description, settings_pane_topstories_options_sponsored, settings_pane_highlights_body2 based on Bug 1404890? Can I remove those?